hedera-node/cli-clients/build.gradle.kts

```diff
@@ -15,7 +15,7 @@
  */
 
 plugins {
-    id("com.hedera.hashgraph.conventions")
+    id("com.hedera.gradle.services")
```

Comment:
Speaking of `cli-clients` - over at the issue this comment https://github.com/hashgraph/hedera-services/issues/11237#issuecomment-1970580413 says `cli-clients` won't be published.  I don't know the pros/cons - but why not?  These are useful tools (for developers), though not part of any running production code.  Or is _that_ the distinction?

Reply:
The project is different in the sense that it does not provide a "library" following clear rules in what is packaged and which dependencies are published (and which module-info is published).
I agree that we can think about publishing this If we would do so, we have to think about the setup. Should it really be fat Jars? Or better follow the same approach as everywhere else to publish Jars with dependencies? How do we expect users to use this?
I'll take this with me to discuss as a follow up.